### PR TITLE
Add Che Theia plugin version builded from master

### PR DESCRIPTION
**Pull request by evidolob**
Signed-off-by: Yevhen Vydolob <yvydolob@redhat.com>

### What does this PR do?
Add Che Theia plugin version `master` builded from Theia master and all Che extensions branch

### CODE REVIEW

File: plugins/org.eclipse.che.editor.theia/master/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+id: org.eclipse.che.editor.theia
+version: master
+type: Che Editor
+name: theia-ide
+title: Eclipse Theia for Eclipse Che
```
Comment (benoitf) [inline plugins/org.eclipse.che.editor.theia/master/meta.yaml]: Eclipse Theia development version ?

Comment (evidolob) [reply plugins/org.eclipse.che.editor.theia/master/meta.yaml]: Done

**Comment by benoitf** [comment]
need to wait travis ^^

**Comment by evidolob** [comment]
@benoitf Travis build passed, should I merge this pull now?

**Comment by benoitf** [comment]
yes you can merge

**Pull request merged**